hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancer.java

```diff
@@ -903,4 +933,33 @@ public String toString() {
     ITERATION_INTERRUPTED,
     CAN_NOT_BALANCE_ANY_MORE
   }
+
+  /**
+   * Sends refresh usage command to trigger all the datanodes to
+   * refresh disk usage info immediately.
+   */
+  private void sendRefreshUsageCommandToAllDNs() {
```

Comment:
It is better to add this as API in NodeManager. The API can take the DN types as input.